aten/src/ATen/native/cuda/GridSampler.cu

```diff
@@ -296,14 +296,14 @@ namespace {
 // lies relative to the entire tensor, so we pass the base grad_input.data and full offset information,
 // including batch * channel offset (NC_offset).
 
-  template <typename scalar_t, typename index_t>
+  template <typename scalar_t, typename index_t, bool input_requires_grad>
```

Comment:
Nit. This could just be a `const` parameter. The performance is going to be similar and it would save us the need to compile the kernel twice.

Reply:
Thanks, changed to `const` parameter. It also makes the code a bit simpler.